### ISSUE TITLE
Create new tag for Typescript users

### ISSUE THREAD

**Issue by LukyVj** [nitin42/terminal-in-react#96]
Hey thanks for this project, 

However, it seems that with the latest version available on NPM, we still cannot use "descriptions" as the `index.d.ts` still contains the typo mentioned here: https://github.com/nitin42/terminal-in-react/issues/68

That would be amazing, really. Thanks.

**Comment by retr00exe** [comment]
Yup, I have the same problem too. It's kinda annoying to edit `index.d.ts` file on node module and change the typescript definition from "description" to "descriptions" :/
